ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -249,6 +250,32 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
     }
   };
 
+  private idResetTap?: number;
+
+  // When tapped 5 time activate the debug mode of the application.
+  // If more than two seconds pass between taps, the counter is reset
+  private onTapAppVersion = () => {
```

Comment:
If you prefer, but it is pretty the same, you could avoid using clearTimeout with [debounce](https://lodash.com/docs/4.17.15#debounce) function

```typescript
private onTapAppVersion = () => {
    if (this.props.isDebugModeEnabled) {
      // do nothing
      return;
    }
    const taps = this.state.tapsOnAppVersion + 1;
    if (taps === tapsToActivateDebugMode) {
      this.props.setDebugModeEnabled(true);
      return;
    }
    this.setState({ tapsOnAppVersion: taps }, () => {
      this.debounceCounterReset();
    });
  };

  private debounceCounterReset = debounce(() => {
    this.setState({ tapsOnAppVersion: 0 });
  }, resetCounterTimeout);
```

Reply:
The problem with the debounce function is that I could not find a way to cancel it in `componentWillUnmount`. Without cancellation, the debounce function keep update the state of the component while unmounted generating a warning. 
I found this behavior in the pinpad and I wanted to avoid to introduce any other potential warnings.

Reply:
it Sounds as a valid point!

Reply:
@fabriziofff anyway for the future: [debounce](https://lodash.com/docs/4.17.15#debounce) comes with **_cancel_** method
I didn't know it before now. I'm going to merge this PR ;)